@types/ApiClient.d.ts

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+type RequestOptions = {
+    headers?: Record<string, string>;
+    body?: Record<string, string>;
```

Comment:
```suggestion
    body?: Record<string, string> | string | Array<unknown>;
```

other options:
* backgroundTask
* timeout
* followRedirects (default to true)
*  maxRetries
* backoffType
* backoffInterval